From d6bb90dea19846eaf4e29851003d09762e70abf2 Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Tue, 12 May 2020 19:27:43 +0900
Subject: [PATCH] edge_spi_dts

add missed places for dts SPI NOR FLASH for EDGE 
---
 arch/arm/dts/rk3399-khadas-edge-u-boot.dtsi | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm/dts/rk3399-khadas-edge-u-boot.dtsi b/arch/arm/dts/rk3399-khadas-edge-u-boot.dtsi
index a7039d74..21d3b0df 100644
--- a/arch/arm/dts/rk3399-khadas-edge-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-khadas-edge-u-boot.dtsi
@@ -10,8 +10,24 @@
 	chosen {
 		u-boot,spl-boot-order = "same-as-spl", &sdhci, &sdmmc;
 	};
+	
+	aliases {
+	    spi0 = &spi1;
+	};
+	
 };
 
 &vdd_log {
 	regulator-init-microvolt = <950000>;
 };
+
+&spi1 {
+    max-freq = <104000000>;
+    status = "okay";
+
+    spiflash: flash@0 {
+	compatible = "winbond,w25q128fw", "jedec,spi-nor";
+	reg = <0>;
+	spi-max-frequency = <104000000>;
+    };
+};
-- 
2.17.1

